pyproject.toml

```diff
@@ -13,5 +13,115 @@ exclude = '''
 '''
 
 [build-system]
-requires = ["setuptools>=41.0", "wheel"]
+requires = ["setuptools>=61", "wheel"]
 build-backend = "setuptools.build_meta"
+
+[project]
+name = "egon.data"
+version = "2.0.0"
+description = "The data used in the eGo^N project along with the code importing, generating and processing it."
+requires-python = ">=3.10"
+license = { text = "AGPL-3.0-or-later" }
+authors = [
+  { name = "eGo^N team", email = "regon-project@rl-institut.de" },
+  { name = "Jonathan Amme" },
+  { name = "Clara Büttner" },
+  { name = "Ilka Cußmann" },
+  { name = "Julian Endres" },
+  { name = "Carlos Epia" },
+  { name = "Kilian Helfenbein" },
+  { name = "Stephan Günther" },
+  { name = "Ulf Müller" },
+  { name = "Amélia Nadal" },
+  { name = "Guido Pleßmann" },
+  { name = "Francesco Witte" },
+]
```

Comment:
This list of authors is now defined here and in the AUTORS.rst which is read by read the docs. 
For release 1.0.0 we said that we want to define them only once, so I changed it. Is it possible to also read the AUTORS.rst file here? 
If that is too complicated, it would be also fine for me to just refer to the "reGon team"